scala/scala.bzl

```diff
@@ -90,7 +90,7 @@ def _build_nosrc_jar(ctx, buildijar):
   mkdir -p {out}_tmp
   # copy any resources
   {cp_resources}
-  {java} -jar {jar} -m {manifest} {out}
+  {java} -jar {jar} -m {manifest} {out} {out}_tmp
```

Comment:
what does this do? I think the name isn't clear enough. Maybe also add a comment